mimic/canned_responses/auth.py

```diff
@@ -24,9 +25,41 @@ def format_timestamp(dt):
     return dt.strftime('%Y-%m-%dT%H:%M:%S.999-05:00')
 
 
+def impersonator_user_role(id, name):
```

Comment:
There should be a ticket for getting rid of this and replacing it with a more "real" role collection in the identity model.